nixos/modules/services/hardware/handheld-daemon.nix

```diff
@@ -0,0 +1,43 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+with lib; let
+  cfg = config.services.handheldDaemon;
+in
+{
+  options.services.handheldDaemon = {
+    enable = mkEnableOption "Enable Handheld Daemon";
+
+    user = mkOption {
+      type = types.str;
```

Comment:
Would it make sense to add some kind of default?

Reply:
should default to the user of the system if possible

Reply:
I don't know that there's a good default.  If there's no way to get the current user, forcing people to specify seems better than making a guess (like Steam Deck's `deck`) that could be wrong for many users.